doc/source/dev.rst

```diff
@@ -94,34 +96,43 @@ as a `remote` in git: ::
 
  git remote add upstream https://github.com/sunpy/sunpy.git
 
-To stay up to date you can grab the latest changes to the SunPy master using
+To stay up to date regularly grab the latest changes to the SunPy master using
 the commands: ::
 
- git pull upstream master
+ git fetch upstream master
+ git merge upstream/master
 
-This will merge the upstream code automatically with your code so you don't need to worry
-about it overwriting your changes. After running either of these commands,
-your local copy of your personal repo is just a copy of the main repo.
-This is the same procedure that you will use in the future to keep yourself synchronised with the
-main repo. To make sure everything is setup correctly, let's make some changes
-to our personal local repo and push those to our personal repo on GitHub. Go ahead and modify one
-of the files, or create a new file (and then run :command:`git add`).
+This will merge the upstream code with your code so you don't
+need to worry about it overwriting your changes. Next let's test this by making
+a change to SunPy. First (always do this!) create a new branch to contain your
+change.
 
-Commit and push the changes to GitHub: ::
+ git checkout -b test
+
+Go ahead and modify one of the files, or create a new file
+(and then run :command:`git add`).
+
+Commit and push the changes to your GitHub: ::
 
  git commit -a -m "My first commit"
  git push
 
-You local repo is now synced with GitHub and ahead of the main repo as it contains
-your personal contribution. Remember to commit after you've done a unit of work (i.e.
-often). This will make it easier for you (in the future) and everyone else to understand
-what you are doing. Also make sure to make your commit statements clear and understandable.
+You local repo is now synced with GitHub and ahead of the main repo as it
+contains your change. Remember to commit after you've done a unit of work (i.e.
+often). This will make it easier for you (in the future) and everyone else to
```

Comment:
I think this sets up the wrong idea. We do not want PRs with each time a file or line change to be committed.